actor.go

```diff
@@ -0,0 +1,515 @@
+// wasmcloud platform core actor
+package actor
+
+import (
+	"errors"
+
+	core "github.com/wasmcloud/interfaces/core/tinygo"
```

Comment:
@stevelr this will need to be fixed in the generator...its been manually added here